jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants.properties

```diff
@@ -34,6 +34,9 @@ Process_Name=Process Name
 Process_Instance_Id=Process Instance Id
 Process_Definition_Id=Process Definition Id
 Process_Instance_Details=Process Instance Details
+Process_Instance_Correlation_Key=Process instance correlation key
```

Comment:
Just because this String is used as the column header I would suggest to drop the _instance_ and use only _Process Correlation Key_